lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,57 @@
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+import { entitlementApi } from './endpoints';
+
+const HEADERS = {
+  Accept: 'application/json',
+  'Content-Type': 'application/json',
+  'X-CSRFToken': Cookies.get('csrftoken'),
+}
+
+export function requestEntitlements({username}) {
```

Comment:
Nit: Yeah, gonna be that person again - these exports should be grouped at the end of the file.

Check out the [`eslint-plugin-import#groups-exports`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/group-exports.md) rule.

The [`airbnb/javascript` style-guide doesn't explicitly mention this](https://github.com/airbnb/javascript#modules) but in [`airbnb/javascript#1275`](https://github.com/airbnb/javascript/issues/1275) one of the maintainer of the style-guide who is also one of the maintainers of `eslint-plugin-import` said
> As far as positioning, there will soon be a linter rule in eslint-plugin-import that will require that all export statements are grouped together - so I'd recommend grouping them together and putting them at the bottom of the file.